.github/workflows/asciidoctor-ghpages.yml

```diff
@@ -0,0 +1,33 @@
+name: GitHub Pages Publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, main ]
```

Comment:
I think we just use `main`

Reply:
This is taken verbatim from https://github.com/manoelcampos/asciidoctor-ghpages-action/blob/main/.github/workflows/asciidoctor-ghpages.yml